### PR TITLE
two changes to enable Boardwalk to run in dev mode

### PR DESCRIPTION
**Pull request by mjkrause**
To install the deployment for feature/commons in `dev` mode, these two changes became necessary after the Boardwalk dashboard service was moved to the `azul` repo:
* instead of cloning from `cgp-dashboard-service` 
* after each repo is cloned, the `check_for_repo` method will checkout the feature branch `feature/commons`

### CODE REVIEW

File: install_bootstrap [hunk 1]
```diff
@@ -40,6 +40,9 @@ MSG
 function check_for_repo {
     if [ ! -d "$2" ]; then
         git clone "$1" "$2"
+	cd "$2"
+	git checkout feature/commons
+	cd ..
```
Comment (coverbeck) [inline install_bootstrap]: Please align with line 42 so the whole block is indented.

**Review by coverbeck — CHANGES_REQUESTED** [review]
Your commit message should reflect the change being made so somebody can do a git log and get a sense of what the commit is without having to look at the code.

See this https://medium.com/@preslavrachev/what-s-with-the-50-72-rule-8a906f61f09c.

It is hard to craft a message using those guidelines, especially the first line, but it is worth it in the future, when you have to go back and look at changes.

File: install_bootstrap [hunk 2]
```diff
@@ -218,7 +221,7 @@ while [[ "${run_boardwalk^^}" != 'Y' &&  "${run_boardwalk^^}" != 'N' ]] ; do
       dcc_dashboard_port=8080
 
       if [ -f boardwalk_launcher_config/boardwalk.config ] ; then
-	  source <(jq -r "to_entries|map(\"\(.key)=\(.value|tostring)\")|.[]" boardwalk_launcher_config/boardwalk.config)
+          source <(jq -r "to_entries|map(\"\(.key)=\(.value|tostring)\")|.[]" boardwalk_launcher_config/boardwalk.config)
```
Comment (coverbeck) [inline install_bootstrap:224]: You should not change this line unless you mean to. I think your editor is converting tabs to spaces or something like that. Please revert this change.

**Review by coverbeck — CHANGES_REQUESTED** [review]
Your comment is much better, but did you see the link I sent you where the first line should max out at 50 and subsequent lines at 72?

File: install_bootstrap (hunk 2 above)
Comment (mjkrause) [reply install_bootstrap:224]: I changed it back. Although I believe that the entire file has space except line 221.

Comment (mjkrause) [reply install_bootstrap:224]: I mean, my aim was only to make it consistent...

Comment (coverbeck) [reply install_bootstrap:224]: It's not changed back, but I suppose you can leave it.

While it's the right change to make, combining it with unrelated changes in the same commit is confusing, because looking back at this will assume this was change was necessary to get Boardwalk running in dev mode.

But it's small enough to just leave. 

But as a general practice, you shouldn't include unrelated formatting changes; or if you do, at least call them out in the commit message.

Comment (mjkrause) [reply install_bootstrap:224]: Okay, I'll follow this going forward. Yes, I saw your comment, and slacked you a histogram of all the length of all commit messages in that repo. That looks very different from the one in the link.

Comment (coverbeck) [reply install_bootstrap:224]: It doesn't look like we're following it. I'm on a one person crusade to change it. :)

I guess there is no SOP on this though, so I probably went overboard, and you can skip.

But I do think your messages should be informative, even if they don't follow the format. and it is informative now.

**Review by coverbeck — APPROVED** [review]
Please squash the commits.
